assets/src/components/ladder.tsx

```diff
@@ -40,113 +45,119 @@ const Ladder = ({
   selectedVehicleId,
 }: Props) => {
   const dispatch = useContext(DispatchContext)
-  const timepointSpacingY =
-    (HEIGHT - MARGIN_TOP_BOTTOM * 2) / (timepoints.length - 1)
+  const [height, setHeight] = useState(500)
 
   const orderedTimepoints: Timepoint[] =
     // Use slice to make a copy of the array before destructively reversing
     ladderDirection === LadderDirection.OneToZero
       ? timepoints.slice().reverse()
       : timepoints
 
+  const timepointSpacingY: number =
+    (height - MARGIN_TOP_BOTTOM * 2) / (timepoints.length - 1)
   const timepointStatusY = timepointStatusYFromTimepoints(
     timepoints,
     timepointSpacingY
   )
+
   const { ladderVehicles, widthOfLanes } = ladderVehiclesFromVehicles(
     vehicles,
     ladderDirection,
     timepointStatusY
   )
 
   const width = 120 + 2 * widthOfLanes
-  // (0, 0) is in the center of the first timepoint
-  const viewBox = [-width / 2, -MARGIN_TOP_BOTTOM, width, HEIGHT].join(" ")
+
+  const ref = createRef<SVGSVGElement>()
```

Comment:
Should this be `useRef` instead of `createRef`? https://stackoverflow.com/questions/54620698/whats-the-difference-between-useref-and-createref

Reply:
That's interesting, I didn't know that distinction. But `createRef` is what we want because we want a reference to the svg, not to the Ladder component, so we create the ref and attach it to the svg.